CONTRIBUTING.md

```diff
@@ -1,122 +1,111 @@
 
-To fix formatting errors, run the following:
-
-```sh
-yarn lint --fix
-```
+> Run `yarn check-js` to validate codestyle
 
-Remember to add tests for your change if possible. Run the unit tests by:
+### iOS
 
-```sh
-yarn test
-```
+1. Open the `react-native-mediapipe/example/ios/MediapipeExample.xcworkspace` file with Xcode
+2. Change signing configuration to your developer account
+3. Select your device in the devices drop-down
+4. Hit run
 
-### Commit message convention
+> Run `yarn check-ios` to validate codestyle
 
-We follow the [conventional commits specification](https://www.conventionalcommits.org/en) for our commit messages:
+### Android
 
-- `fix`: bug fixes, e.g. fix crash due to deprecated method.
-- `feat`: new features, e.g. add new method to the module.
-- `refactor`: code refactor, e.g. migrate from class components to hooks.
-- `docs`: changes into documentation, e.g. add usage example for the module..
-- `test`: adding or updating tests, e.g. add integration tests using detox.
-- `chore`: tooling changes, e.g. change CI config.
+1. Open the `example/android/` folder with Android Studio
+2. Start the metro bundler in the `example/` directory using `yarn start`
+3. Select your device in the devices drop-down
+4. Once your device is connected, make sure it can find the metro bundler's port:
+   ```
+   adb reverse tcp:8081 tcp:8081
+   ```
+6. Hit run
 
-Our pre-commit hooks verify that your commit message matches this format when committing.
+> Run `yarn check-android` to validate codestyle
 
-### Linting and tests
+### Docs
 
-[ESLint](https://eslint.org/), [Prettier](https://prettier.io/), [TypeScript](https://www.typescriptlang.org/)
+1. Edit the relevant file, it may be easiest to search for what you're editing to find the right file
+2. Install all dependencies by running `yarn` inside the `docs` folder
 
-We use [TypeScript](https://www.typescriptlang.org/) for type checking, [ESLint](https://eslint.org/) with [Prettier](https://prettier.io/) for linting and formatting the code, and [Jest](https://jestjs.io/) for testing.
+> Run `yarn start` to generate the docs, you can then view them in your browser to confirm your changes
 
-Our pre-commit hooks verify that the linter and tests pass when committing.
+## Committing
 
-### Publishing to npm
+### Codestyle
 
-We use [release-it](https://github.com/release-it/release-it) to make it easier to publish new versions. It handles common tasks like bumping version based on semver, creating tags and releases etc.
+Great code produces great products. That's why we love to keep our codebases clean, and to achieve that, we use linters and formatters which output errors when something isn't formatted the way we like it to be.
 
-To publish new versions, run the following:
+Before pushing your changes, you can verify that everything is still correctly formatted by running all linters:
 
-```sh
-yarn release
+```
+yarn check-all
```

Comment:
```
yarn typecheck
yarn lint
```

Reply:
FYI, if you want to see mostly what you can `yarn`, its in `package.json` "scripts" section